docs/supported_versions.md

```diff
@@ -0,0 +1,206 @@
+          dated 20240826 or newer before updating OCP.
+        name: SRIOVFailedToConfigureVF
+        url: https://issues.redhat.com/browse/NHE-1171
+```
+
+## Specifying the OCP release in HostedCluster Custom Resource
+`.spec.release` dictates the version of the control plane.
+
+The HostedCluster propagates the intended `.spec.release` to the `HostedControlPlane.spec.release` and runs the appropriate Control Plane Operator version.
+
+The HostedControlPlane orchestrates the rollout of the new version of the Control Plane components along with any OCP component in the data plane through the new version of the [cluster version operator (CVO)](https://github.com/openshift/cluster-version-operator). This includes resources like:
+
+- the CVO itself
+- cluster network operator (CNO)
+- cluster ingress operator
+- manifests for the kube API-server (KAS), scheduler, and manager
+- machine approver
+- autoscaler
+- infra resources needed to enable ingress for control plane endpoints (KAS, ignition, konnectivity, etc.)
+
+Traditionally, in standalone OCP, the CVO has been the sole source of truth for upgrades. In hosted OCP cluster, the responsibility is currently split between Control Plane Operator (CPO) and CVO. This enabled the flexibility and speed the HyperShift project needed for the CPO to support the management/guest cluster topology and the multiple customizations needed for manifests that otherwise would have needed to be segregated in the payload.
+
+Using the information from the `status.version.availableUpdates` and `status.version.conditionalUpdates` described in the previous section, you can set the `.spec.release` in the HostedCluster custom resource to start the control plane upgrade.
+
+## NodePools
+`.spec.release` dictates the version of any particular NodePool.
+
+A NodePool will perform a Replace/InPlace rolling upgrade according to `.spec.management.upgradeType`. See [NodePool Upgrades](./automated-machine-management/nodepool-lifecycle.md#upgrades-and-data-propagation) for details. Using the available upgrades information from the `status.version.availableUpdates` described in the previous section, you can set the `.spec.release` in the NodePool custom resources to start the data plane upgrade.
+
+## Upgrading Hosted Clusters via MCE Console
+In the MCE console, select `All clusters` view at the top and navigate to Infrastructure -> Clusters to view managed hosted clusters with the `Upgrade available` link. You can also click on a managed hosted cluster to view the cluster details which also shows the `Upgrade available` link. You can update the control plane and nodepools by clicking the link. The list of the available updates is not from the hosted cluster's CVO and can be different than the available updates from the HostedCluster's `status.version.availableUpdates` and `status.version.conditionalUpdates`. Therefore, referring to the hosted cluster's available and conditional updates in the status is recommended.
```

Comment:
We should probably require the user to refer to the hosted cluster's available and conditional updates since if they don't then they break their cluster. Also mention their cluster will break if they select an update that's not supported.